src/routes/landing/components/Heading.js

```diff
@@ -3,9 +3,7 @@
 import React from 'react'
 
 import LocationBig from '../assets/LocationBig.svg'
-import type { TFunction } from 'react-i18next'
 import { withNamespaces } from 'react-i18next'
```

Comment:
Component should stay connected and this should remain here, right?

Reply:
You can remove it as it is no longer used. Should be easy to add again afterwards.